nexus/db-queries/src/db/saga_recovery.rs

```diff
@@ -162,16 +171,10 @@ where
     }))
 }
 
-// Creates new page params for querying sagas.
-fn new_page_params(
-    marker: Option<&uuid::Uuid>,
-) -> DataPageParams<'_, uuid::Uuid> {
-    DataPageParams {
-        marker,
-        direction: dropshot::PaginationOrder::Ascending,
-        limit: std::num::NonZeroU32::new(100).unwrap(),
-    }
-}
+const BATCH_SIZE: NonZeroU32 = unsafe {
```

Comment:
I'd use [SQL_BATCH_SIZE](https://github.com/oxidecomputer/omicron/blob/1e3e9674a32c8770b266ec725e3ec6966d9da48d/nexus/db-queries/src/db/datastore/mod.rs#L138-L144) here.  This didn't exist when the old code was written but now that we have it I think we may as well use it here.

Reply:
Updated